### ISSUE TITLE
RFP - unofficial-Google-Chat 

### ISSUE THREAD

**Issue by dtantono** [chocolatey-community/chocolatey-package-requests#1310]
Checklist

- [x] The package I am requesting does not already exist on https://chocolatey.org/packages;
- [x] There is no open issue for this package;
- [x] The issue title starts with 'RFP - ';
- [x] The download URL is public and not locked behind a paywall / login;

## Package Details

Software project URL :https://github.com/squalou/google-chat-linux
Direct download URL for the software / installer : https://github.com/squalou/google-chat-linux/releases/download/5.15.17-1/google-chat-linux_Setup_5.15.17-1.exe
Software summary / short description:
unofficial Google chat client wrapped in electron

**Comment by pauby** [comment]
The title of whatever package is created should reflect that is unofficial. My concern is that if there is an official google-meet package created in future, it will clash.

This _may_ also cause some issues (take from https://github.com/squalou/google-chat-linux#readme):

> There will be "SmartScreen" warning about how unsafe this .exe is, Windows pretending it has detected something nasty and is protecting you. I Don't Care and won't buy a certificate.

**Comment by dtantono** [comment]
Is windows code signed app necessary?

**Comment by pauby** [comment]
Not necessarily. But Windows Defender loves picking up on unsigned apps. If it can be silently installed then it should be fine. It is something that needs to be taken into consideration though.
